### ISSUE TITLE
Inset box shadows with spread_radius > border_radius explode

### ISSUE THREAD

**Issue by mstange** [servo/webrender#1916]
https://bugzilla.mozilla.org/show_bug.cgi?id=1411074

[Testcase](https://bug1411074.bmoattachments.org/attachment.cgi?id=8921264)

Expected vs actual results:
<img width="1142" alt="screen shot 2017-10-23 at 7 15 51 pm" src="https://user-images.githubusercontent.com/961291/31917681-c50c314e-b826-11e7-813c-24ab9cb0e3d9.png">

This seems to be caused by the `sharpness_scale` adjustment that's done when computing the impact of the spread radius on the corner radius. This adjustment is in the spec, but neither Firefox nor Chrome implement it, so I think we can just drop it again.

```html
<!DOCTYPE html>
<meta charset="utf-8">

<title>Box-shadows with spread radius needs to affect non-zero border-radius</title>

<style>

body {
  margin: 0;
}

div {
  display: inline-block;
  width: 80px;
  height: 80px;
  margin: 20px;
  box-shadow: 0 0 0 10px inset blue;
}
.withRadius {
  border-radius: 20px;
}

</style>

<div style="border-radius: 20px"></div>
<div style="border-radius: 25px"></div>
<div style="border-radius: 10px"></div>
<div style="border-radius: 9px"></div>
<div style="border-radius: 8px"></div>
<div style="border-radius: 7px"></div>
<div style="border-radius: 6px"></div>
<div style="border-radius: 5px"></div>
<div style="border-radius: 4px"></div>
```

**Comment by mstange** [comment]
This was already fixed in #1889.

**Comment by mstange** [comment]
I was wrong.
